src/app/Main/Body/QueryPage/Results/Charts/QueryPageContent-BarChart.component.tsx

```diff
@@ -64,6 +65,26 @@ function processingTableDataToApexData(
     };
 }
 
+function generateAxisLabels(
+    t: TFunction,
+    rowName: string,
+    colName: string
+): [string, string] {
+    let lableXaxis = t("Results.TableHead");
+    let lableYaxis = "";
+    if (!_.isEmpty(rowName)) {

```

Comment:
I try to omit using external libraries whenever possible. So here ``if(rowName !== "")`` would do the same thing. But if you like using lodash you can keep it because it is a very common library.

Reply:
True, in this case it is probably a bit too much to use Lodash.